### PR TITLE
media: Transfers features from manager to sfu

### PR DESCRIPTION
**Pull request by geirbakke**
### Description

This transfers any feature prefixed by sfu to the sfu server

### CODE REVIEW

File: packages/media/src/webrtc/VegaRtcManager.ts [hunk 1]
```diff
@@ -256,6 +256,9 @@ export default class VegaRtcManager implements RtcManager {
             roomName: this._room.name,
             eventClaim: this._room.isClaimed ? this._eventClaim : null,
             lowBw: "true",
+            ...Object.keys(this._features || {})
+                .filter((featureKey) => this._features[featureKey] && /^sfu/.test(featureKey))
```
Comment (havardholvik) [inline packages/media/src/webrtc/VegaRtcManager.ts:260]: Do we need the check for `this._features[featureKey]`?

**Review by havardholvik — APPROVED** [review]

**Review by pnts-se-whereby — APPROVED** [review]
LGTM

File: packages/media/src/webrtc/VegaRtcManager.ts (hunk 1 above)
Comment (geirbakke) [reply packages/media/src/webrtc/VegaRtcManager.ts:260]: just for filtering out features that are false/off

**Pull request merged**